docs/src/introduction.md

```diff
@@ -5,23 +5,110 @@
-pure-Rust Patina components, and contributing to the ecosystem - all without assuming prior experience with
-Patina itself.
+To put it simply, Patina is a [UEFI spec](https://uefi.org/specifications) compliant pure Rust DXE Core with an
+interface for writing monolithically compiled [Patina components](component/getting_started.md) (drivers) using a
+dependency injection model. It should be noted that Patina takes a more opinionated stance on some implementation
+details to provide a more secure runtime, so we are not fully PI spec compliant. Due to this, there are some key
+platform requirements that differ from a non-Patina implementation. See [Patina Requirements](integrate/patina_dxe_core_requirements.md)
+for more information regarding this.
 
-Before getting started, you may want to read the [Patina Background](patina.md), which outlines the project's goals
-and design philosophy.
+**This book is the "catch-all" for Patina:** it contains various categories of documentation regarding Patina that can
+be broken up into these 5 categories:
 
-In addition, here are some of the more commonly referenced documentation in this book:
+1. Background of Patina and it's goals.
+2. Integrating the Patina code into your platform.
+3. Developing Patina components.
+4. Contributing to Patina.
+5. Development guidance and standards that apply to both Patina and Patina component developers.
 
-1. [Patina Background](patina.md)
-2. [RFC Lifecycle](rfc_lifecycle.md)
-3. [Platform Integration](integrate/patina_dxe_core_requirements.md)
-4. [Component Development](component/getting_started.md)
-5. [Developer Guides](dev/documenting.md)
+If you have not read this book before, we suggest you start with **(1) [Patina Background](patina.md)**, which outlines
+the project's goals and design philosophy.
+
+## Important Crates and Repositories
+
+While the Patina project maintains a large list of distinct crates throughout our various repositories (See
+[below](#all-repositories)), these are almost entirely logical separation of code to support proper decoupling and
+faster build times. If you are just starting with Patina, the project has two main crates that you should review:
+
+**1. [patina_dxe_core](https://crates.io/crates/patina_dxe_core):** The library crate containing the Patina DXE Core,
+which is customized and extended for each individual platform.
+
+**2. [patina](https://crates.io/crates/patina):** The SDK for working with Patina, whether you are creating a Patina
+component or developing the Patina DXE core.
+
+In addition to this, there are two additional repositories you should review, which are specific to integrating Patina
+as a platform's DXE Core:
+
+**1. [patina-dxe-core-qemu](https://github.com/OpenDevicePartnership/patina-dxe-core-qemu):** A repository containing
+two distinct implementations / configurations of the `patina_dxe_core` library crate for our Q35 and SBSA virtual
```

Comment:
```suggestion
two distinct configurations of the `patina_dxe_core` library crate for our Q35 and SBSA virtual
```
I would say these are not implementations of the patina_dxe_core library crate, they are simply configurations of it. They are implementations of a platform specific DXE core.